Makefile

```diff
@@ -135,20 +123,6 @@ test-integration: test-integration-prepare
 	cd tests && \
 		behave --show-timings -D skip_setup --junit
 
-.PHONY: clean
-clean:
-	@echo 'Cleaning up'
-
-	rm -f version.txt
-	rm -rf build
-	rm -rf dist
-	rm -rf *.egg-info
-	rm -f debian/files
-	rm -rf debian/mdb-ch-tools*
-	rm -f ../mdb-ch-tools_*
-	find . -name __pycache__ -type d -exec rm -rf {} +
```

Comment:
Why is it deleted? And if we want to delete it, help text below should be updated too.